app/src/main/res/values/strings.xml

```diff
@@ -24,4 +24,6 @@
     <string name="error_lenght_from_0_to_100">Limit value must be from 0 to 100</string>
     <string name="must_not_empty">Must not empty</string>
     <string name="error_unusable_characters">There are unusable characters</string>
+    <string name="a">a</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.a` appears to be unused
 Unused resources make applications larger and slow down builds.